napalm/junos/junos.py

```diff
@@ -1823,6 +1823,38 @@ def ping(self, destination, source=C.PING_SOURCE, ttl=C.PING_TTL,
 
         return ping_dict
 
+    def _get_root(self):
+        """get root user password."""
+        _DEFAULT_USER_DETAILS = {
+            'level': 0,
```

Comment:
why don't you just set this number to 15 here and avoid later the `user_details.update({'level': 15})`?

Reply:
right... was just copy pasting...
fixed in recent commit 32e5661